examples/acs2/handeye/utils.py

```diff
@@ -0,0 +1,42 @@
+def calculate_performance(hand_eye, population):
+    """
+    Analyzes all possible transition in maze environment and checks if there
+    is a reliable classifier for it.
+    :param hand_eye: maze object
+    :param population: list of classifiers
+    :return: percentage of knowledge
```

Comment:
Z tego co widzę to zwracany jest `dict` z kilkoma metrykami. Warto opisać co każda z nich oznacza w praktyce